tools/operator/testdata/kafka-leveldb-notls/samplejs-input.yml

```diff
@@ -0,0 +1,102 @@
+tls: false
```

Comment:
I assume you are going to remove this line from all the files in a different PR...

Reply:
yup